RecoTracker/LSTCore/src/alpaka/NeuralNetwork.h

```diff
@@ -29,8 +30,10 @@ namespace lst::t5dnn {
                                                    float (&output)[OUT_FEATURES],
                                                    const float (&weights)[IN_FEATURES][OUT_FEATURES],
                                                    const float (&biases)[OUT_FEATURES]) {
+#pragma unroll OUT_FEATURES
     for (unsigned int i = 0; i < OUT_FEATURES; ++i) {
       output[i] = biases[i];
+#pragma unroll IN_FEATURES
```

Comment:
are both similarly useful? I'd think the innermost is more significant

Reply:
It seems the benefit is coming from the outermost pragma mainly.

Only innermost pragma
<img width="1406" alt="Screenshot 2024-10-14 at 8 17 31 AM" src="https://github.com/user-attachments/assets/e73309a0-fb67-4e6b-a569-c59762724a56">

Only outermost pragma
<img width="1406" alt="Screenshot 2024-10-14 at 8 22 06 AM" src="https://github.com/user-attachments/assets/9f73fc76-d109-4773-b722-a8aa98c32abb">